### PR TITLE
Clarify TAO processing when it comes to revalidation responses

### LINKED ISSUE #201: Specify TAO check for 304 responses
In these responses, the TAO header to be used must be the one from the original resource. Maybe it is worth clarifying this on the spec since it may require implementors to store these (Chrome has a special field for it). Also, make sure there is sufficient test coverage for these cases.

### PR DESCRIPTION
**Pull request by yoavweiss**
Closes #201

***
<a href="https://pr-preview.s3.amazonaws.com/yoavweiss/resource-timing/pull/206.html" title="Last updated on Apr 24, 2019, 6:50 PM UTC (396be56)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/resource-timing/206/7f033b2...yoavweiss:396be56.html" title="Last updated on Apr 24, 2019, 6:50 PM UTC (396be56)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -964,9 +964,9 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
       data-cite="FETCH#concept-request-window">window</a>'s
       <a data-cite="HTML/webappapis.html#concept-settings-object-origin">origin</a>,
       continue to the next <var>request</var>.</p></li>
-      <li><p>If the <a>Timing-Allow-Origin</a> header value list
-      does not contain a value which is byte-for-byte identical to
-      the <a
+      <li><p>If the <var>request</var>'s response' <a>Timing-Allow-Origin</a>
```
Comment (npm1) [inline index.html]: Wouldn't it be "request's response's"? Maybe simplify to "If the TAO header value list of the request's response..."

File: index.html [hunk 2]
```diff
@@ -979,6 +979,11 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 a cross-origin resource followed by same-origin resources without an explicit
 opt-in, TAO will pass while CORS will fail. Future iterations of this
 specification may attempt to unite those definitions.</p>
+<p class=note>The Timing-Allow-Origin header arrive as part of a cached
```
Comment (npm1) [inline index.html]: may arrive? Or what do you mean when you say it always arrives as part of a cached response?

File: index.html (hunk 1 above)
Comment (yoavweiss) [reply index.html]: 👍

File: index.html (hunk 2 above)
Comment (yoavweiss) [reply index.html]: Added a "may"

**Comment by yoavweiss** [comment]
@npm1 PTAL

File: index.html [hunk 3]
```diff
@@ -979,6 +978,11 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 a cross-origin resource followed by same-origin resources without an explicit
 opt-in, TAO will pass while CORS will fail. Future iterations of this
 specification may attempt to unite those definitions.</p>
+<p class=note>The Timing-Allow-Origin header may arrive as part of a cached
+response.  In case of cache revalidation, according to
```
Comment (npm1) [inline index.html]: Nit: two spaces before "In"

Comment (yoavweiss) [reply index.html]: ack

**Pull request merged**